lib/reducers/create-otp-reducer.js

```diff
@@ -51,11 +51,21 @@ function createOtpReducer (config, initialQuery) {
             query: clone(state.currentQuery),
             pending: true,
             planResponse: null,
+            type: 'ITINERARY',
```

Comment:
The request type could be included in the action and then the PROFILE_REQUEST wouldn't be needed.